crates/transaction-emitter/src/cluster.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
I think we can implement this logic without needing to introduce cluster and instance which are Cluster-test specific concepts.

Reply:
My thought is, for those generic usage of tx_emitter, cluster and instance are more like unified types of objects to hold necessary information with different size of scope

Reply:
My thought is, for all those generic usages of tx_emitter only without via Forge, cluster and instance are more like two general/unify types of objects hold necessary information with distinct size of scopes.

Reply:
ah got it, i guess i needed to look a bit harder.